### DISCUSSION TITLE
Meaning of NvChad logo

### DISCUSSION THREAD

**Discussion by trongthanh** [NvChad/NvChad#3010] [General]
Anyone curious about meaning of the NvChad logo?

<img width="235" alt="image" src="https://github.com/user-attachments/assets/0f5e58f6-c0f2-4fcb-8b38-3a697da4f919">

I asked ChatGPT and here's the answer:

> [T]his graphic is written in Arabic and it says: “العلم قبل القول والعمل”, which translates to: “Knowledge comes before speech and action.”
>
> It emphasizes the importance of acquiring knowledge before taking action or speaking. This concept aligns with wisdom often found in Islamic teachings, highlighting the value of learning as a foundation for all endeavors.
>
> In the context of NvChad, a Neovim configuration framework, this phrase might represent the idea that understanding and knowledge of tools are essential before using them effectively.

**Comment by siduck** [comment]
yes thats right
